astropy/io/votable/tree.py

```diff
@@ -3445,6 +3613,26 @@ def type(self, type):
             vo_raise(E18, type, self._config, self._pos)
         self._type = type
 
+    @property
+    def mivot_block(self):
+        """
+        Returns the XML MIVOT block serialized as a string.
+        If the host resource is of type results, it is taken from the first
+        child resource with a MIVOT block, if any.
+        Otherwise, it is taken from the host resource
+        """
+        if self.type == "results":
+            for resource in self.resources:
+                if str(resource._mivot_block).strip() != "":
+                    return resource._mivot_block
```

Comment:
This seems to be returning an actual `MivotBlock` instance rather than a string as indicated in the function doc string.  Although the `MivotBlock` can be treated as a string in many ways, should the doc string be adjusted to say that the return is a `MivotBlock`?